pkg/persistence/config/loader/config_entry_loader.go

```diff
@@ -70,6 +74,8 @@ func parseConfigEntry(
 			continue
 		}
 
+		report.GetReporterFromContextOrDiscard(ctx).ReportLoading(report.StateSuccess, nil, "config validated", &result.Coordinate)
```

Comment:
One more thing: The message might get shown to customers in the app. I recommend that we properly capitalize all of them.
E.g. instead of `config validated` -> `Successfully validated configuration`.

Reply:
Decision: we get rid of this message